bears/haskell/HaskellLintBear.py

```diff
@@ -19,7 +23,12 @@ class HaskellLintBear:
     """
 
     LANGUAGES = {'Haskell'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='hlint')}
+    REQUIREMENTS = {AnyOneOfRequirements(
+        [CabalRequirement(package='hlint', version='1.9.35'),
```

Comment:
identation
this is better:
```
REQUIREMENTS = {AnyOneOfRequirements(
        [CabalRequirement(package='hlint', version='1.9.35'),
         DistributionRequirement(apt_get='hlint')
         ]
    ),
}
```

Reply:
@manankalra , i don't see any issue with this, as it still follows PEP8.

Reply:
@manankalra , that is worse!

@ankit01ojha , no it is not following PEP8,

`{AnyOneOfRequirements(` is ok, but then all of the parts on subsequent lines would need to be aligned with the opening bracket, which is the parentheses (*round* bracket), not the brace `{`. 

I told you how to do it at https://github.com/coala/coala-bears/pull/2195#discussion_r179915624

Anyway, the issue has been re-assigned to someone else, as you are busy with GSoC at @openMF and we need this urgently.